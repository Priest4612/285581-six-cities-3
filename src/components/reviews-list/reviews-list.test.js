import React from 'react';
import renderer from 'react-test-renderer';

import ReviewsList from './reviews-list.jsx';


const reviews = [{
  id: 3,
  authorAvatar: `angelina`,
  authorName: `Angelina`,
  rating: 2.8,
  description: `123 3rfg432 54t4566 56457 5436er5 6456`,
  dateTime: `25-02-2020`,
  idOffer: 101,
}, {
  id: 4,
  authorAvatar: `max`,
  authorName: `Max`,
  rating: 4.8,
  description: `123 3rfg432 54t4566 56457 5436er5 6456`,
  dateTime: `27-02-2020`,
  idOffer: 101,
}, {
  id: 5,
  authorAvatar: `angelina`,
  authorName: `Angelina`,
  rating: 3.8,
  description: `123 3rfg432 54t4566 56457 5436er5 6456`,
  dateTime: `23-02-2020`,
  idOffer: 202,
}];

describe(`Checked OfferDetails`, () => {
  it(`Render OfferDetails`, () => {
    const tree = renderer
      .create(<ReviewsList
        reviews={reviews}
      />)
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
