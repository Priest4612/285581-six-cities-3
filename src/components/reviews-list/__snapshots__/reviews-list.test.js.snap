// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Checked ReviewsList Render ReviewsList 1`] = `
Array [
  <h2
    className="reviews__title"
  >
    Reviews Â·
    <span
      className="reviews__amount"
    >
      3
    </span>
  </h2>,
  <ul
    className="reviews__list"
  >
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height="54"
            src="img/avatar-angelina.jpg"
            width="54"
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Angelina
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "56%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          123 3rfg432 54t4566 56457 5436er5 6456
        </p>
        <time
          className="reviews__time"
          dateTime="25-02-2020"
        >
          25-02-2020
        </time>
      </div>
    </li>
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height="54"
            src="img/avatar-max.jpg"
            width="54"
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Max
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "96%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          123 3rfg432 54t4566 56457 5436er5 6456
        </p>
        <time
          className="reviews__time"
          dateTime="27-02-2020"
        >
          27-02-2020
        </time>
      </div>
    </li>
    <li
      className="reviews__item"
    >
      <div
        className="reviews__user user"
      >
        <div
          className="reviews__avatar-wrapper user__avatar-wrapper"
        >
          <img
            alt="Reviews avatar"
            className="reviews__avatar user__avatar"
            height="54"
            src="img/avatar-angelina.jpg"
            width="54"
          />
        </div>
        <span
          className="reviews__user-name"
        >
          Angelina
        </span>
      </div>
      <div
        className="reviews__info"
      >
        <div
          className="reviews__rating rating"
        >
          <div
            className="reviews__stars rating__stars"
          >
            <span
              style={
                Object {
                  "width": "76%",
                }
              }
            />
            <span
              className="visually-hidden"
            >
              Rating
            </span>
          </div>
        </div>
        <p
          className="reviews__text"
        >
          123 3rfg432 54t4566 56457 5436er5 6456
        </p>
        <time
          className="reviews__time"
          dateTime="23-02-2020"
        >
          23-02-2020
        </time>
      </div>
    </li>
  </ul>,
]
`;
